noir-projects/aztec-nr/aztec/src/discovery/private_logs.nr

```diff
@@ -68,7 +70,7 @@ pub unconstrained fn do_process_log<Env>(
 
     let (log_type_id, log_metadata, log_content) = decode_log_plaintext(log_plaintext);
 
-    if log_type_id == 0 {
+    if log_type_id == PRIVATE_NOTE_LOG_TYPE_ID {
```

Comment:
There's a comment above this line that mentions the constants 0 and 1 that we'd want to update.